examples/nylas-connect/app.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+import json
+import sys
+import os
+import flask
+import requests
+import urllib
+import logging
+import subprocess
+
```

Comment:
I would like a big comment here explaining the program flow --- where the user starts, how she's redirect to Google, etc. You can use asciiflow (http://asciiflow.com/) for this.